app/scripts/models/reliers/oauth.js

```diff
@@ -187,7 +187,7 @@ define([
       var self = this;
       var clientId = self.get('clientId');
 
-      return self._oAuthClient.getClientInfo(clientId)
+      return self._fxaClient.getOAuthClientInfo(clientId)
```

Comment:
I realize this is outside of the scope of this PR, and I did line 190 to begin with, I feel like passing in the client to the relier model just to get the OAuth client info is ...  not right. But having app-start.js fetch the oauth client info and pass it in doesn't feel right either.